modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -264,6 +264,21 @@ def from_pandas(cls, df, data_cls):
     def from_arrow(cls, at, data_cls):
         return cls(data_cls.from_arrow(at))
 
+    # Dataframe exchange protocol
+
+    def to_dataframe(self, nan_as_null: bool = False, allow_copy: bool = True) -> dict:
```

Comment:
I don't think `to_dataframe()` should be returning a `dict`

Reply:
We can fix that as part of a separate PR.